includes/Webhook/Service/WebhookSubscriptionService.php

```diff
@@ -300,4 +291,50 @@ private function saveSubscription(
 
         update_option(self::WEBHOOK_SETTINGS_PATH, json_encode($array));
     }
+
+    /**
+     * @param  string $apiKey
+     *
+     * @return void
+     * @throws \Exception
+     */
+    public function subscribeToWebhooks(string $apiKey): void
+    {
+        $hooks = AccountSettingsWebhook::ACCOUNT_SETTINGS_WEBHOOKS;
+        $webhookSubscriptionService = new WebhookSubscriptionService();
+
+        foreach ($hooks as $webhookClass) {
+            $service = (new $webhookClass())->setApiKey($apiKey);
+            $webhookCallback = $webhookSubscriptionService->createCallbackUrl($service, 'v1');
```

Comment:
De version is niet superbelangrijk (nu), maar als we veranderen hoe het werkt wel, ik zou hem daarom wat meer exposed maken (bv als constante) niet als een string 'diep'. weggestopt.